.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+If you are reporting a new issue, please first search for a matching issue in
+this repository.
+
+-->
+
+
+**Output of `dotnet --info`:**
```

Comment:
If your not expecting people to build this then I guess it doesnt matter. I was just thinking it would be helpful if an issue with build came in. But likely the issues will all be from consumption so I can remove it.